Support/lib/ruby_utils.rb

```diff
@@ -0,0 +1,55 @@
+module RubyUtils
+  module_function
+
+  # Try to find the Ruby executable `name`. Return an array containing the
+  # command(s) needed to run the executable, or `nil` if the executable could
+  # not be found.
+  #
+  # Typical usage looks like this:
+  #
+  #     Dir.chdir(ENV['TM_PROJECT_DIRECTORY']) if ENV['TM_PROJECT_DIRECTORY']
+  #     executable = RubyUtils.find_executable('rspec')
+  #     if executable
+  #       system(*executable, '--some', '--additional', 'args')
+  #     else
+  #       # Executable not found, so use fallback / display alert / …
+  #     end
+  #
+  # Supports the following cases:
+  #
+  # 1. If an appropriate `TM_*` environment variable is present and points to an
```

Comment:
It started there initially :) But then I realized that the env var is the most “finegrained” setting: I can always add a `.tm_properties` file to a project (or even a subdir) setting this variable to override whatever else setting would take effect normally.

A (slightly contrived) example: I have a project where the Gemfile contains `rubocop`, but for some reason I want to override this and use another version of RuboCop when running it from TextMate. If the env var has the highest priority, I can do so by simply setting `TM_RUBOCOP` in this project’s `.tm_properties` file.

But to be honest, I’m totally not sure about this 🤔 /cc @sorbits